src/graphql/graphql.ts

```diff
@@ -281,8 +283,14 @@ export function paginatedQuery(query: string): PaginatedQuery {
 }
 
 export function paginatedQueryV2(query: string): PaginatedQuery {
-  return process.env.GRAPHQL_V2_PAGINATOR === 'relay' ?
-    paginatedWithRelayV2(query) : paginateWithOffsetLimitV2(query);
+  switch (process.env.GRAPHQL_V2_PAGINATOR) {
+    case 'relay':
+      return paginatedWithRelayV2(query);
+    case 'where':
```

Comment:
I insist - `whered`

Reply:
Actual name is `keyset cursor` if that matters. https://hasura.io/docs/latest/queries/postgres/pagination/#keyset-cursor-based-pagination

Reply:
I'm guessing this was a typo that impacted the prototype?

Reply:
Let's name it keyset

Reply:
fine!

Reply:
nerds ))